b9lab-content/2-what-is-cosmos/02-blockchain-and-cosmos.md

```diff
@@ -0,0 +1,162 @@
+# Blockchain Technology and Cosmos
+
+Begin your journey with this brief review of blockchain technology, how Cosmos came into being, and what it brings to the world:
+
+* An internet of blockchains
+* A better dApp user experience
+* A simplified specialized dAapp development experience
+* Facilitated scalability
+* Increased sovereignty to dApps
+* Speed and fast finality
+
+## The World of Blockchains — From Public General Purpose to Purpose-Built Chains
+
+The building blocks of blockchain technology can be found in the 1980s and 1990s when breakthroughs in computer science and cryptography laid the necessary groundwork. Blockchain technology per se was invented in 2008.
+
+Necessary preliminary work included append-only, cryptographically secure logs using hashing, authentication and encryption keys, as well as the conceptual development of smart contracts and consensus mechanisms in peer-to-peer (P2P) networks with Byzantine fault-tolerance (BFT).
+
+On the fateful October 31, 2008, an individual, or group, calling itself Satoshi Nakamoto proposed a **P2P network for a digital currency**, calling it **Bitcoin**. It introduced a novel consensus mechanism, now referred to as Nakamoto Consensus, that uses Proof-of-Work (PoW) to enable nodes to reach agreement in a decentralized network. Suddenly, it became possible to send online payments directly between parties **independently of financial institutions and trusted third parties**. Bitcoin became the first public, decentralized application.
+
+<HighlightBox type="info">
+
+  Want to take a look at the [Bitcoin White Paper](https://bitcoin.org/bitcoin.pdf)?
+
+</HighlightBox>
+
+<ExpansionPanel title="I heard about PoW, tell me more">
+
+Proof-of-Work (PoW), used to achieve BFT, is best described as a cryptographic puzzle solved by a network's node, called miner. The puzzle is a task of pre-defined, arbitrary difficulty. At the scale of the network, the outcome is akin to a lottery with a single winning node.
```

Comment:
The difficulty of the task is not arbritrary, it is calculated by the latest input of hashpower.

Reply:
```suggestion
Proof-of-Work (PoW), used to achieve BFT, is best described as a cryptographic puzzle solved by a network's node, called miner. The puzzle is a task of pre-defined, arbitrary difficulty. At the current scale of the network, the outcome is akin to a lottery with a single winning node.
```